index.html

```diff
@@ -903,19 +903,89 @@ <h3>Emma Brown</h3>
       </div>
 
   </section>
+  <footer class="container py-4 py-md-5 px-4 px-md-3">
+    <div class="row d-flex justify-content-center align-items-center text-center">
+      <div class="col-md-6 mb-3">
+        <form>
+          <h5>Subscribe to our newsletter</h5>
+          <p>Monthly digest of what's new and exciting from us.</p>
+          <div class="d-flex gap-2 justify-content-center align-items-center">
+            <input id="newsletter1" type="text" class="form-control w-full" placeholder="Email address">
+            <button class="btn btn-primary" type="button">Subscribe</button>
+          </div>
+        </form>
+      </div>
+    </div>
+    <div class="row mt-5">
+      <div class="col-md-4 mb-3">
+        <a class="d-inline-flex align-items-center mb-2 text-body-emphasis text-decoration-none footer-title"
+          href="/" aria-label="Helping-Hands">
+          Helping-Hands
+        </a>
+        <div class="vs">
+        <ul class="list-unstyled small">
+          <li class="mb-2">HelpingHands2.0 connects individuals with local job opportunities, offering job listings, skill development programs, and resources for resume building and interview preparation. It's free for job seekers, allowing them to set up job alerts and contact employers directly, with a strong emphasis on security and data privacy..</li>

```

Comment:
Helping Hands connects individuals with local job opportunities, offering job listings, skill development programs, and resources for preparation.

Reply:
change the text to what I have written